analysis/codelists.py

```diff
@@ -0,0 +1,36 @@
+from ehrql import codelist_from_csv
+
+acute_otitis_media_cod = codelist_from_csv(
+    "codelists/opensafely-pharmacy-first-acute-otitis-media-treatment-dmd.csv",
+    column="code",
+)
+
+first_impetigo_treatment_cod = codelist_from_csv(
```

Comment:
From discussing with Milan I think he missed that the "first" here is from "Pharmacy first" - can remove "first_" to avoid later confusion
```suggestion
impetigo_treatment_cod = codelist_from_csv(
```

Reply:
thanks :) addressed in ca1e1573c933a290905a92e8d8cc6065fe151a77